### PR TITLE
Remove deprecated code

### PR DESCRIPTION
**Pull request by chrfritsch**

### CODE REVIEW

File: harbourmaster.install [hunk 1]
```diff
@@ -9,7 +9,7 @@
  *
  */
 function harbourmaster_install() {
-  drupal_set_message(
+  \Drupal::messenger()->addMessage(
```
Comment (alexpott) [inline harbourmaster.install]: addWarning() and remove the last arg.

**Review by alexpott — CHANGES_REQUESTED** [review]
Also I think that we could remove jQuery.ready() see https://www.drupal.org/project/drupal/issues/3086375

**Comment by chrfritsch** [comment]
> Also I think that we could remove jQuery.ready() see https://www.drupal.org/project/drupal/issues/3086375

TBH I am afraid to do this change since I am not able to test this module.

**Comment by alexpott** [comment]
> TBH I am afraid to do this change since I am not able to test this module.

Fair enough. I don't think jQuery is going to be removed in D9.

**Review by dbosen — APPROVED** [review]
lgtm

**Pull request merged**